roles/openshift_logging/templates/fluentd.j2

```diff
@@ -117,11 +117,11 @@ spec:
         - name: "OPS_COPY_PASSWORD"
           value: "{{ops_copy_password | default('')}}"
         - name: "USE_JOURNAL"
-          value: "{{openshift_logging_fluentd_use_journal|lower}}"
+          value: "{{openshift_logging_fluentd_use_journal|default('')|lower}}"
```

Comment:
If you are specifying the default in defaults, you shouldn't need to use the default filter here.